arangod/RocksDBEngine/RocksDBCollection.h

```diff
@@ -210,6 +211,7 @@ class RocksDBCollection final : public RocksDBMetaCollection {
   bool _cacheEnabled;
   /// @brief number of index creations in progress
   std::atomic<int> _numIndexCreations;
+  arangodb::TransactionStatistics& _statistics;
```

Comment:
Is it guaranteed that the TransactionStatistics object always outlives all RocksDBCollection instances? I think under normal circumstances this is the case, but I am not sure about unit tests/mocks.

Reply:
I am not sure about guarantees. Tests never failed, however. Not much of a proof. One can always break this dependency. I am not sure, what you expect how to fix this.